libs/webb-ui-components/src/components/Modal/ModalContent.tsx

```diff
@@ -44,7 +44,7 @@ export const ModalContent = forwardRef<HTMLDivElement, ModalContentProps>(
           {...props}
           className={twMerge(
             // Prevent the focus border from showing.
-            'focus:outline-none',
+            'outline-none',
```

Comment:
```tsx
focus-visible:outline-none
```

I want to prevent this when the modal is focused by the keyboard.

![CleanShot 2025-02-15 at 02 32 35@2x](https://github.com/user-attachments/assets/106c3658-3eb6-4b89-970d-92948b9de1e2)